lib/knapsack/adapters/minitest_adapter.rb

```diff
@@ -48,7 +48,7 @@ def set_test_helper_path(file_path)
 
       def self.test_path(obj)
         # Pick the first public method in the class itself, that starts with "test_"
-        test_method_name = obj.public_methods(false).select{|m| m =~ /^test_/ }.first
```

Comment:
When you use the previous version of the code with `/^test_/` then test for shoulda-context still passes. It should fail.